website/src/hooks/hubSpot.ts

```diff
@@ -85,16 +85,10 @@ interface HookProps {
 
 const loadHubSpotScript = (): HTMLScriptElement | Element => {
     const hubSpotScript = '//js.hsforms.net/forms/v2.js'
-    const script = document.querySelector(`script[src="${hubSpotScript}"]`)
-
-    if (!script) {
```

Comment:
Removing this causes the `v2.js` `hubSpotScript` to be appended to the head multiple times when navigating back and forth between pages and also if a page has multiple forms.

For pages that include multiple forms of the same formId, we need to render the form in different places but not append the hubSpot script again.